ci/ray_ci/doc/api.py

```diff
@@ -1,5 +1,12 @@
+import re
+
 from enum import Enum
 from dataclasses import dataclass
+from typing import Optional
+
+
+SPHINX_AUTOSUMMARY_HEADER = ".. autosummary::"
+SPHINX_AUTOCLASS_HEADER = ".. autoclass::"
```

Comment:
`_SPHINX_AUTOSUMMARY_HEADER` ? or will this also going to be used outside this file?